style.css

```diff
@@ -0,0 +1,300 @@
+	justify-content: center;
+	align-items: center;
+    align-content: center;
+}
+
+.thirdSection{
+    display: flex;
+    justify-content: center;
+    padding: 40px;
+    
+}
+
+.box1{
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	justify-content: center;
+    padding: 20px;
+}
+
+.box2{
+	
+    padding: 20px;
+
+}
+
+.box3{
+	
+    padding: 20px;
+
+}
+
+.box4{
+	
+    padding: 20px;
+
+}
+
+.button2 {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+    box-shadow: 0 10px 30px rgba(0, 0, 0, 0.4);
+    background-color: black;
+    color: white;
+    width: 300px;
+    height: 50px;
+    justify-content: space-around;
+    font-size: 20px;
+
+}
+
+#firstSection {
+
+width: 960px;
+margin: auto;
+display: flex;
+flex-direction: column;
+flex-wrap: nowrap;
+justify-content: center;
+align-items: center;
+align-content: center;
+text-align: center;
+background-color: rgba(232,217,217,.3);
+height: 450px;
+}
+
+#firstParagraph {
+    width: 430px;
+    text-align: justify;
+    text-align-last: center;
+}
+
+#amazing {
+    font-size: 50px;
+    font-weight: 1000;
+}
+.hexagon {
+    color: yellow;
+    width: 200px;
+    margin: auto;
+    flex-direction: column;
+flex-wrap: nowrap;
+justify-content: center;
+align-items: center;
+align-content: center;
+display: block;
+padding: 50px;
+}
+
+form{
+    background-color: rgba(0,0,0,.05);
+    float: left;
+    width: 600px;
+    justify-content: left;
+    
+}
+
+.searchPackages{
+    padding: 5px;
+}
+
+
+.search{
+    border: none;
+    width: 560px;
+background: none;
+position: relative;
+height: 40px;
+padding-left: 10px;
+font-size: 16px;
+}
+
+
+.planjoin{
+    display: flex;
+    justify-content: space-between;
+width: 400px;
+
+}
+
+h7 {
+  background-color: rgba(255,204,53,.4);  
+  width: 700px;
+  text-align: center;
+  transform: skew(9deg,0deg);
+  font-size: 1.9rem;;
+  font-weight: bolder;
+  font-style: italic;
+
+}
+
+.SeePlans{
+    background-color: red;
+    color: white;
+    width: 180px;
+    height: 60px;
+    box-shadow: 8px 8px 0 rgba(251,59,73,.2);
+    font-weight: 1000;
+    font-size: 20px;
+}
+
+.JoinForFree{
+    background-color: white;
+width: 180px;
+height: 60px;
+font-weight: 1000;
+border: none;
+font-size: 20px;
+
+}
+
+.join{
+text-decoration: none;
+color: black;
+border:1px solid gray;  
+width: 50px;
+text-align: center;
+
+}
+
+
+.login{
+    text-decoration: none;
+    color: black;
+}
+
+input{
+    outline: 0;
+    
+}
+
+.magnglass{
+   padding-left: 5px;
+   padding-top: 10px;
+   float: inline-end;
+}
+
+
+p5{
+width: 650px;
+padding: 30px;
+text-align: center;
+font-size: 17px;
+}
+
+
+.zero-configuration{
+flex-direction: row;
+width: 40px;
+justify-content: flex-start;
+float: left
+
+}
+
+.team-management{
+    flex-direction: row;
+    width: 40px;  
+}
+
+.familiar-features{
+    flex-direction: row;
+    width: 40px;  
+
+}
+
+.npm-audit{
+    flex-direction: row;
+    width: 40px;  
+    
+}
+
+
+
```

Comment:
In this case, you can make your code shorter by writing it like this `h1, h2, h3, h4 {color: red; font-size:15px;}`